packages/edge-login-ui-rn/src/components/themed/EdgeTextFieldOutlined.js

```diff
@@ -121,291 +146,288 @@ const CornerBorder = ({
+  const {
+    // Content options:
+    error,
+    label: placeholder = '',
+
+    // Appearance:
+    isClearable = false,
+    marginRem,
+    size = 'big',
+    showSearchIcon = true,
+
+    // Callbacks:
+    onBlur,
+    onChangeText,
+    onClear,
+    onFocus,
+
+    // Other React Native TextInput properties:
+    value = '',
+    ...inputProps
+  } = props
+
+  const [containerHeight, setContainerHeight] = useState(0)
+
+  // animation
+  const inputRef = useRef<TextInput>(null)
+  const placeholderMap = useSharedValue(
+    value ? ANIMATION_STATES.FOCUSED : ANIMATION_STATES.INIT
+  )
+  const placeholderSize = useSharedValue(ANIMATION_STATES.INIT)
+  const containerWidth = useSharedValue(ANIMATION_STATES.INIT)
+  const colorMap = useSharedValue(ANIMATION_STATES.INIT)
+
+  // input methods
+  const focus = () => inputRef.current && inputRef.current.focus()
+  const blur = () => inputRef.current && inputRef.current.blur()
+  const isFocused = () =>
+    Boolean(inputRef.current && inputRef.current.isFocused())
+  const clear = () => {
+    Boolean(inputRef.current && inputRef.current.clear())
+    if (onChangeText != null) onChangeText('')
+  }
 
-    // styles
-    const theme = useTheme()
-    const styles = getStyles(theme)
-    const spacings = sidesToMargin(
-      mapSides(fixSides(marginRem, 0.5), theme.rem)
-    )
-    const placeholderSpacerWidthAdjust = theme.rem(1)
-    const {
-      inactiveColor,
-      activeColor,
-      errorColor,
-      fontSize,
-      placeholderSpacerAdjust,
-      placeholderScale,
-      placeholderSizeScale,
-      paddingVertical,
-      inputStyles,
-      placeholderTextStyles,
-      placeholderPaddingStyles,
-      placeholderSpacerPaddingStyles,
-      inputContainerStyles,
-      prefixStyles,
-      suffixStyles,
-      hintLeftMargin
-    } = getSizeStyles(theme, size, showSearchIcon)
-
-    const errorState = useCallback(
-      () => error !== null && error !== undefined,
-      [error]
-    )
+  // styles
+  const theme = useTheme()
+  const styles = getStyles(theme)
+  const spacings = sidesToMargin(mapSides(fixSides(marginRem, 0.5), theme.rem))
+  const placeholderSpacerWidthAdjust = theme.rem(1)
+  const {
+    inactiveColor,
+    activeColor,
+    errorColor,
+    fontSize,
+    placeholderSpacerAdjust,
+    placeholderScale,
+    placeholderSizeScale,
+    paddingVertical,
+    inputStyles,
+    placeholderTextStyles,
+    placeholderPaddingStyles,
+    placeholderSpacerPaddingStyles,
+    inputContainerStyles,
+    prefixStyles,
+    suffixStyles,
+    hintLeftMargin
+  } = getSizeStyles(theme, size, showSearchIcon)
 
-    const handleFocus = e => {
-      placeholderMap.value = withTiming(ANIMATION_STATES.FOCUSED)
-      if (!errorState()) colorMap.value = withTiming(ANIMATION_STATES.FOCUSED)
-      focus()
-      if (onFocus) onFocus(e)
-    }
+  const errorState = useCallback(() => error != null, [error])
```

Comment:
Suggestion: `const isErrorState = error != null && error !== ''`?

Reply:
To be fixed in a future PR, perhaps. For now it "works", even if it's not great.